packages/webpack/src/plugin.js

```diff
@@ -69,6 +69,9 @@ class LavaMoatPlugin {
     if (!options.policyLocation) {
       options.policyLocation = path.join('.', 'lavamoat', 'webpack')
```

Comment:
note that `path.join('.', ...anything)` is equivalent to `path.join(...anything)`